daemons/dss-s3-copy-sfn/app.py

```diff
@@ -0,0 +1,16 @@
+import os
+import sys
+
+import domovoi
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'domovoilib'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import dss.stepfunctions.s3copyclient as s3copyclient
+import dss.stepfunctions.generator as generator
+
+app = domovoi.Domovoi()
+
+
+transformer = generator.Transformer(app)
+transformer.transform(s3copyclient.sfn)
```

Comment:
Any chance we could use more descriptive names than generator and transformer?

Reply:
Done.